packages/product-components/src/components/SelectAssetModal/SelectAssetModal.tsx

```diff
@@ -1,29 +1,23 @@
-import { useMemo, useState } from 'react';
+import { ReactNode, useState } from 'react';
 import { useIntl } from 'react-intl';
 
-import {
-    AssetLogo,
-    Column,
-    Icon,
-    Input,
-    NewModal,
-    useScrollShadow,
-    VirtualizedList,
-} from '@trezor/components';
+import { Column, NewModal, useScrollShadow, VirtualizedList } from '@trezor/components';
 import { mapElevationToBackgroundToken, spacings } from '@trezor/theme';
-import { getNetworkByCoingeckoId, Network } from '@suite-common/wallet-config';
+import { NetworkSymbol } from '@suite-common/wallet-config';
 
 import { AssetItem } from './AssetItem';
-import { NetworkTabs } from './NetworkTabs';
 import { AssetItemNotFound } from './AssetItemNotFound';
 
 export interface SelectAssetOptionCurrencyProps {
     type: 'currency';
-    value: string; // CryptoId (networkId + contractAddress)
-    label: string; // token shortcut
-    cryptoName: string | undefined; // full name
-    coingeckoId: string; // CryptoId (networkId)
-    contractAddress?: string; // CryptoId (contractAddress)
+    symbol: string;
+    label?: string;
+    balance?: string;
+    badge?: string | ReactNode;
```

Comment:
`string` is assignable to `ReactNode`
```suggestion
    badge?: ReactNode;
```